src/theory/ext_theory.cpp

```diff
@@ -84,9 +84,9 @@ bool ExtTheoryCallback::getReduction(int effort,
 ExtTheory::ExtTheory(ExtTheoryCallback& p,
                      context::Context* c,
                      context::UserContext* u,
-                     OutputChannel& out)
+                     InferenceManagerBuffered& im)
```

Comment:
Is there a reason why this is InferenceManagerBuffered? Can't this just be TheoryInferenceManager?

Reply:
I guess it can. Both arithmetic and strings were using inference managers derived from `InferenceManagerBuffered`, and I didn't really think about this... I'll change it.